### PR TITLE
03.09.22 release

### PR DESCRIPTION
**Pull request by essiequoi**
# What does this PR do?

creates release notes for 0.8.2 - 0.8.4

### CODE REVIEW

**Review by ematta — DISMISSED** [review]
We need to add https://linear.app/gridai/issue/GRID-8566

**Comment by essiequoi** [comment]
@ematta @oojo12 i added the piece about the dependency file path fix. pls review and let me know if this is good to post

File: changelog/2022-03-09.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+
+## :wrench: March 9, 2022
+
+**CLI version: 0.8.4**
+
+:heavy_check_mark: Resolves an issue where using a relative path for the `dependency_file_info` property in a Run config was breaking. For example, this now works:
```
Comment (rlizzo) [inline changelog/2022-03-09.md:6]: i would add "this now works if you were operating from a subdirectory of a git repo" or something analogous.

File: changelog/2022-03-08.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+
+## :wrench: March 8, 2022
```
Comment (rlizzo) [inline changelog/2022-03-08.md:2]: why are these duplicated?

File: changelog/2022-03-09.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+:heavy_check_mark: Runs will fail more quickly if there is an issue with image building.
+
+:heavy_check_mark: Resolves issue with `--num_trials` parameter being ignored.
+
+:heavy_check_mark: Logging improvements to silence noisy stacktraces.
+
+:heavy_check_mark: 'pytorch' and 'torch' are now both equal and acceptable inputs to the framework option for `grid run` 
+
+ex: `--framework pytorch` == `--framework torch`
```
Comment (rlizzo) [inline changelog/2022-03-09.md:28]: didn't we already announce this stuff?

**Review by rlizzo — CHANGES_REQUESTED** [review]
see code comments. I don't know why the changelog is duplicated either?

File: changelog/2022-03-08.md (hunk 2 above)
Comment (essiequoi) [reply changelog/2022-03-08.md:2]: ahh it shouldn't be. i updated the filename from 03.08 to 03.09 and it just created a new version of the file but didn't delete. will fix.

File: changelog/2022-03-09.md (hunk 3 above)
Comment (essiequoi) [reply changelog/2022-03-09.md:28]: i did not - looks like these were all merges made since last Tuesday, and I haven't made any announcements since then

Comment (essiequoi) [reply changelog/2022-03-09.md:28]: there was a bug where "pytorch" was not working as an option and that was fixed and merged this week

**Review by rlizzo — APPROVED** [review]
LGTM

**Pull request merged**